style.css

```diff
@@ -0,0 +1,128 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+body {
+  margin: 0;
+  padding: 0;
+  background: #e5e5e5;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+}
+
+.main {
+  background: url('./images/header-background-shape-desktop.svg');
+  min-height: 100vh;
+  width: 100vw;
+  margin: 0;
+  padding-top: 72px;
+}
+
+.toolbar {
+  position: fixed;
+  margin: 0;
+  height: 72px;
+  width: 95vw;
+  padding: 0 24px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background: #fff;
+}
+
+.logo-wrapper {
+  height: 20px;
+  width: 76px;
+}
+
+.menu-icon {
+  height: 16px;
+  width: 17px;
+}
+
+.header {
+  background: url('./images/Header-bg.svg');
+  background-size: contain;
+  background-repeat: no-repeat;
+  height: 85vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.banner {
+  width: 63.888%;
+}
+
+.banner h1 {
+  font-size: 48px;
+  font-weight: 700;
+  line-height: 60px;
+  letter-spacing: 0.3700000047683716px;
+  text-align: left;
+}
+
```

Comment:
**REQUIRED:** It is impossible to see your name on phone mode, the toolbar is on it. Please kindly fix this issue. Use inspector to see how your project look like on different sizes.
![image](https://user-images.githubusercontent.com/77872355/129120166-f267d18d-cd39-4019-ad6a-eca9d68948a2.png)